### PR TITLE
#U54 fix: improves arrow transition for duration 

### PR DESCRIPTION
**Pull request by chrisostome01**
# PR Description

This pull request aims to reduce the arrow transition during the animation 

# Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you  Make sure you check every completed task. list them in this style

- [x]  The animation duration of the arrow is reduced and smoothened
- [x]  The arrow movement is reduced

# Description of tasks that were expected to be completed and that are not yet completed

- [x]  `none`

# How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependencies 
2. run `npm run dev`
3. hover over any link.

# Screenshots (If appropriate/applicable)

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by IbrahimBagalwa — APPROVED** [review]
## Great Job @sezeranoJchrisostome  👍 

LGTM 👍

File: tailwind.config.js [hunk 1]
```diff
@@ -128,7 +130,8 @@ module.exports = {
         "3xl": "1px 1px 14px rgba(0, 0, 0, 0.1)",
       },
       transitionDuration: {
-        '0.4': '0.4s',
+        0.2: '0.2s',
```
Comment (serapieTuyishime) [inline tailwind.config.js]: You should use the recommended tailwindcss way of using 200 or 400 instead of 0.2 and 0.4

**Review by serapieTuyishime — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (chrisostome01) [reply tailwind.config.js]: yeah sure let me go with that.

**Review by serapieTuyishime — APPROVED** [review]
Good job Mate

**Review by Yvad60 — APPROVED** [review]
Looks good to me 
Approved!

**Pull request merged**